packages/client/src/Call.ts

```diff
@@ -879,14 +883,15 @@ export class Call {
       const startedAt = callState?.startedAt
         ? Timestamp.toDate(callState.startedAt)
         : new Date();
+      const pins = callState?.pins || [];
```

Comment:
```suggestion
      const pins = callState?.pins ?? [];
```

As suggested by sonarcloud